components/Newsletter.js

```diff
@@ -44,11 +44,17 @@ const Newsletter = ({ tags, placement }) => {
           { signupError &&
             <div className="error">{ signupError }</div>
```

Comment:
In this design, error should be above, and also let's use `error-box` class so its styled.

Reply:
I placed it above. Let me know if it is ok.